autosubmit/platforms/sgeplatform.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, expid, name, config):
         self.job_status['QUEUING'] = ['qw', 'hqw', 'hRwq', 'Rs', 'Rts', 'RS', 'RtS', 'RT', 'RtT']
         self.job_status['FAILED'] = ['Eqw', 'Ehqw', 'EhRqw', 's', 'ts', 'S', 'tS', 'T', 'tT', 'dr', 'dt', 'dRr', 'dRt',
                                      'ds', 'dS', 'dT', 'dRs', 'dRS', 'dRT']
-        self._pathdir = "\$HOME/LOG_" + self.expid
+        self._pathdir = "\\$HOME/LOG_" + self.expid
```

Comment:
I think this platform file is being deleted in #2349 . If that happens before this one is merged, we need to pay attention to remove this file when fixing conflicts :+1: